[Docs] missing `build:demos` in package.json

## Bug report

### Steps to reproduce
1. View https://github.com/material-components/material-components-web/blob/master/CONTRIBUTING.md#building-mdc-web
2. run `npm i`
3. run `npm run build:demos` 

### Actual behavior

```
npm run build:demos
npm ERR! missing script: build:demos
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     build:min

npm ERR! A complete log of this run can be found in:
```

### Expected behavior

work as what `CONTRIBUTING.md` say:

`npm run build:demos # Cleans out build/ and builds demo CSS/JS files, e.g. for deploying to App Engine`

### Screenshots

<!-- If applicable, add screenshots to help explain your issue. -->

### Your Environment:

<!-- please complete the following information -->

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | 0c95c9f7bf1e0d465e99fd7dd3f1497d37d871ff |
| Browser          | no browser, command line only |
| Operating System | Ubuntu 20.04.1, Linux 5.8.0-38-generic |

### Additional context
<!-- Add any other context about the problem here. -->

### Possible solution
<!-- Add any other context about the problem here. -->

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -64,7 +64,6 @@ You can find much more information with respect to building components within ou
 npm run build # Cleans out build/ and builds unminified files for each MDC Web package
 npm run build:min # Builds minified files for each MDC Web package
 npm run dist # Runs both of the above commands sequentially
-npm run build:demos # Cleans out build/ and builds demo CSS/JS files, e.g. for deploying to App Engine
 ```
 
 ### Linting / Testing / Coverage Enforcement

